### PR TITLE
Fix AppData

### LINKED ISSUE #1: Missing on Flathub.org
Tor Browser Launcher is missing on the Flathub.org website for some reason. It is showing fine in GNOME Software.

### LINKED ISSUE #2: Add OARS rating to AppData
There are currently no [OARS](https://hughsie.github.io/oars/) metadata in the Tor Browser Launcher AppData file. It would be nice adding them.

### PR DESCRIPTION
**Pull request by AsciiWolf**
Closes #2 and hopefully also fixes #1.

### CODE REVIEW

File: fix-AppData-file.patch [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+index 5577868..f4b6d67 100644
 --- a/share/metainfo/torbrowser.appdata.xml
 +++ b/share/metainfo/torbrowser.appdata.xml
 @@ -1,7 +1,7 @@
  <?xml version="1.0" encoding="UTF-8"?>
  <!-- Copyright 2014 Micah Lee <micah@micahflee.com> -->
- <component>
+-<component>
 - <id>org.torproject.torbrowser.desktop</id>
++<component type="desktop">
```
Comment (bilelmoussaoui) [inline fix-AppData-file.patch]: I would use a desktop-application component here.

File: fix-AppData-file.patch [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 + <id>torbrowser.desktop</id>
```
Comment (bilelmoussaoui) [inline fix-AppData-file.patch]: As you're already patching the appdata file, any reason for not setting the right id here?

Comment (AsciiWolf) [reply fix-AppData-file.patch]: Upstream don't use the _com.github.micahflee.torbrowser-launcher_ id and I am sending all patches upstream + would like to drop them in next releases.

File: fix-AppData-file.patch (hunk 1 above)
Comment (AsciiWolf) [reply fix-AppData-file.patch]: Oops, you're right, thanks!

Comment (bilelmoussaoui) [reply fix-AppData-file.patch]: See https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html

>  The desktop-application component type is the same as the desktop component type - desktop is the older type identifier for desktop-applications and should not be used for new metainfo files, unless compatibility with very old AppStream tools (pre 2016) is still wanted.

**Pull request merged**